gencode.c

```diff
@@ -6363,20 +6363,17 @@ gen_check_802_11_data_frame(compiler_state_t *cstate)
  * against Q_IP and Q_IPV6.
  */
 static struct block *
-gen_proto(compiler_state_t *cstate, bpf_u_int32 v, int proto, int dir)
```

Comment:
So I think that we included the direction in order to provide future proofing against BPF code that would be able to distinguish between incoming and outgoing packets.